ThunderCloud/SpotlightListItemCell.swift

```diff
@@ -7,15 +7,152 @@
 //
 
 import UIKit
+import ThunderBasics
 import ThunderTable
 
-public class SpotlightImageCollectionViewCell: UICollectionViewCell {
+public class CarouselCollectionViewLayout: UICollectionViewFlowLayout {
+    
+    public override var itemSize: CGSize {
+        get {
+            return CGSize(
+                width: (collectionView?.bounds.width ?? UIScreen.main.bounds.width) -
```

Comment:
How does this work when in a split-view controller? I guess it's the same as other items where it does correctly scale? :)

Reply:
`.collectionView` should **always** be non-nil, I just put it in as an absolute worst-case scenario :D